website/docs/r/management_group_template_deployment.html.markdown

```diff
@@ -0,0 +1,137 @@
+---
+subcategory: "Template"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_management_group_template_deployment"
+description: |-
+  Manages a Template Deployment at a Management Group Scope.
+---
+
+# azurerm_management_group_template_deployment
+
+Manages a Template Deployment at a Management Group Scope.
+
+~> **Note:** Deleting a Deployment at the Management Group Scope will not delete any resources created by the deployment. 
+
+~> **Note:** Deployments to a Management Group are always Incrementally applied. Existing resources that are not part of the template will not be removed.
+
+## Example Usage
+
+```hcl
+resource "azurerm_management_group_template_deployment" "example" {
+  name                  = "example"
+  location              = "West Europe"
+  management_group_name = "example"
+  template_content      = <<TEMPLATE
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "policyAssignmentName": {
+      "type": "string",
+      "defaultValue": "[guid(parameters('policyDefinitionID'), resourceGroup().name)]",
+      "metadata": {
+        "description": "Specifies the name of the policy assignment, can be used defined or an idempotent name as the defaultValue provides."
+      }
+    },
+    "policyDefinitionID": {
+      "type": "string",
+      "metadata": {
+        "description": "Specifies the ID of the policy definition or policy set definition being assigned."
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Authorization/policyAssignments",
+      "name": "[parameters('policyAssignmentName')]",
+      "apiVersion": "2019-09-01",
+      "properties": {
+        "scope": "[subscriptionResourceId('Microsoft.Resources/resourceGroups', resourceGroup().name)]",
+        "policyDefinitionId": "[parameters('policyDefinitionID')]"
+      }
+    }
+  ]
+}
+TEMPLATE
+
+  parameters_content = <<PARAMS
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "policyDefinitionID": {
+      "value": "/providers/Microsoft.Authorization/policyDefinitions/0a914e76-4921-4c19-b460-a2d36003525a"
+    }
+  }
+}
+PARAMS
+}
+
+// NOTE: whilst we show an inline template here, we recommend
+// sourcing this from a file for readability/editor support
```

Comment:
arguably this'd make more sense for an example here?

Reply:
I'll admit I copy-pasted from the example in the existing deployment resource containing this comment, I'll add one via the file mechanism also.